### PR TITLE
chore(eda): Use the gif picture in README

### LINKED ISSUE #226: Use the gif picture in README
This is a reminder issue to change the static picture in README into a gif picture.

### PR DESCRIPTION
**Pull request by dylanzxc**
Resolve #226

# Description

Use the gif picture in README also add two examples of using plot(df,x) with categorical and numerical columns

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have already squashed the commits and make the commit message conform to the project standard.
- [ ] I have already marked the commit with "BREAKING CHANGE" or "Fixes #" if needed.
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,19 +40,27 @@ tasks with a single function call.
 
 * Want to understand the distributions for each DataFrame column? Use `plot`.
 
-<center><a href="https://sfu-db.github.io/dataprep/eda/introduction.html#analyzing-basic-characteristics-via-plot"><img src="https://github.com/sfu-db/dataprep/raw/develop/assets/plot(df).png"/></a></center>
+<center><a href="https://sfu-db.github.io/dataprep/eda/introduction.html#analyzing-basic-characteristics-via-plot"><img src="assets/plot(df).gif"/></a></center>
```
Comment (dovahcrow) [inline README.md:43]: Hi please use the absolute URL instead of the relative one. PyPI will not show things correctly with relative url.

Comment (dylanzxc) [reply README.md:43]: The GIF won't be displayed if I used the absolute URL here. I'm also curious.

Comment (dovahcrow) [reply README.md:43]: Change the URL to this https://github.com/dylanzxc/dataprep/raw/test/assets/plot(df).gif

Change it to back sfu-db before we merge.

Comment (dylanzxc) [reply README.md]: fixed

Comment (jnwang) [reply README.md]: Can you guys take a look at the issue with <center>? 

https://pypi.org/project/dataprep/

Screen Shot 2020-07-14 at 10 47 42 PM

Comment (dovahcrow) [reply README.md]: @dylanzxc Can you also remove the center HTML tag?

Comment (dylanzxc) [reply README.md]: removed. Do you know why the center HTML tag affects the pypi now? The reason is HTML5 removed <center>?

Comment (dovahcrow) [reply README.md]: I'm not sure. Maybe you can find some clue in their renderer? https://github.com/pypa/readme_renderer

**Comment by dovahcrow** [comment]
rendered: https://github.com/dylanzxc/dataprep/tree/test

**Comment by dovahcrow** [comment]
This looks nice to me!

**Pull request merged**